pkg/action/upgrade.go

```diff
@@ -391,6 +391,9 @@ func (u *Upgrade) releasingUpgrade(c chan<- resultMessage, upgradedRelease *rele
 	}
 
 	if u.Wait {
+		u.cfg.Log(
+			"waiting for release %s resources (created=%d, updated=%d, deleted=%d)",
```

Comment:
I only have one nit (otherwise it looks good). Here you have an `=` between the thing being counted and the count. I was looking elsewhere in our log message and I didn't see a case where an `=` is being used. Instead there is either a space or `: ` (colon and a space). For the sake of consistency, can you switch to one of these.

Reply:
Nit but good point for consistency